rpc/dataconn/timeoutconn/timeoutconn_readv_unsupported.go

```diff
@@ -0,0 +1,11 @@
+// +build illumos solaris
+
+package timeoutconn
+
+import "net"
+
+func (c Conn) readv(buffers net.Buffers) (n int64, err error) {
+	// Go does not expose the SYS_READV symbol for Solaris / Illumos - do they have it?
```

Comment:
illumos certainly has `readv()`, just go does not yet know how to use the system call.

Reply:
Yeah, I guess someone could open a Go issue about that, but I'd like to defer that until zrepl users on illumos complain about performance.